operator/README.md

```diff
@@ -189,8 +189,8 @@ $ kubectl -n bundle-test patch serviceaccount default -p '{"imagePullSecrets": [
 # Use one-liner above.
 
 # 4. Run bundle.
-$ bin/operator-sdk-1.14.0 run bundle \
-  docker.io/stackrox/stackrox-operator-bundle:v$(make --quiet tag) \
+$ bin/operator-sdk-1.22.2 run bundle \
+  quay.io/rhacs-eng/stackrox-operator-bundle:v$(make --quiet tag) \
```

Comment:
Had to use `rhacs-eng` here because the `stackrox-io/stackrox-operator-bundle` is empty atm.